lib/facter/custom_facts/core/execution/base.rb

```diff
@@ -138,9 +138,9 @@ def log_stderr(msg, command, logger)
           logger.debug(format(STDERR_MESSAGE, command, msg.strip))
         end
 
-        def builtin_command?(command)
-          output, _status = Open3.capture2("type #{command}")
-          output.chomp =~ /builtin/ ? true : false
+        # optional function to be defined in subclass if necessary
+        def builtin_command?(_command_)
+          nil
```

Comment:
Maybe keep `false` for consistency?  Also the "Ruby way" for unused variable is to name them `_variable`.

Overall LGTM.